docs/jobs/qgis_installation_finder.md

```diff
@@ -0,0 +1,85 @@
+# QGIS installation finder
+
+Use this job to find installed QGIS version for automatic definition of QDT environnement variable `QDT_QGIS_EXE_PATH` needed for shortcut creation
+
+----
+
+## Compatibility
+
+This job is compatible with:
+
+- Windows
+- Linux
+
+----
+
+## Use it
+
+Sample job configuration in your scenario file:
+
+```yaml
+- name: Find installed QGIS
+  uses: qgis-installation-finder
+  with:
+    version_priority:
+      - "3.36"
```

Comment:
It would be nice to have the ability to defined the preferred version in environment variable: `QDT_PREFERRED_QGIS_VERSION`. Can be added in a next PR.